command/monitor.go

```diff
@@ -373,8 +373,17 @@ func formatAllocMetrics(metrics *api.AllocationMetric, scores bool, prefix strin
 
 	// Print scores
 	if scores {
-		for name, score := range metrics.Scores {
-			out += fmt.Sprintf("%s* Score %q = %f\n", prefix, name, score)
+		if len(metrics.ScoreMetaData) > 0 {
+			for scorer, scoreMeta := range metrics.ScoreMetaData {
+				for _, nodeScore := range scoreMeta {
+					out += fmt.Sprintf("%s* Scorer %q, Node %q = %f\n", prefix, scorer, nodeScore.NodeID, nodeScore.Score)
```

Comment:
From the demo it would be nice if this output was tabular. Sorting by the normalized score first then lexicographical by node id. Example:

```
node                                 | binpack  | node-affinity | normalized |
4ebde21b-daa9-f264-16dd-b6ba88efdd58 | 0.080626 | 1.000000      | 0.540313   |
5f2db925-69d4-e677-d3a9-b166155153c6 | 0.080626 | 1.000000      | 0.540313   |
14ebcc35-34d0-1812-14c9-b4397d603723 | 0.080626 | 0.333333      | 0.206980   |
```